selfdrive/modeld/models/driving.h

```diff
@@ -6,13 +6,16 @@
 #include "cereal/messaging/messaging.h"
 #include "common/modeldata.h"
 #include "common/util.h"
-#include "selfdrive/modeld/models/nav.h"
+#include "selfdrive/modeld/models/commonmodel.h"
+#include "selfdrive/modeld/runners/run.h"
 
 constexpr int FEATURE_LEN = 128;
 constexpr int HISTORY_BUFFER_LEN = 99;
 constexpr int DESIRE_LEN = 8;
 constexpr int DESIRE_PRED_LEN = 4;
 constexpr int TRAFFIC_CONVENTION_LEN = 2;
+constexpr int NAV_FEATURE_LEN = 256;
```

Comment:
It's not ideal that NAV_FEATURE_LEN is specified in both driving.h and nav.h, but this is sort of an artifact of the fact that the output size of navmodel.onnx and the size of the `nav_features` input on supercombo.onnx aren't guaranteed to be the same. These constants should probably be checked against each other in the modeld SConscript, but that seemed outside the scope of this PR to me.